test/src/responder/multiResponder.test.ts

```diff
@@ -46,12 +47,12 @@ describe("MultiResponder", () => {
         const providerMock = mock(ethers.providers.JsonRpcProvider);
         when(providerMock.getNetwork()).thenResolve({ chainId: 1, name: "test" });
         when(providerMock.getTransactionCount("address", "pending")).thenResolve(1);
-        const provider = instance(providerMock);
+        const provider = throwingInstance(providerMock);
 
         const signerMock = mock(ethers.providers.JsonRpcSigner);
         when(signerMock.getAddress()).thenResolve("address");
         when(signerMock.provider).thenReturn(provider);
-        signer = instance(signerMock);
+        signer = throwingInstance(signerMock);
```

Comment:
I think you can add
`when(signerMock.sendTransaction(anything())).thenResolve()`

to mock the sendTransaction